### DISCUSSION TITLE
NEW XIAO ADD-ON WITH SENSORS

### DISCUSSION THREAD

**Discussion by usermctech** [Seeed-Studio/OSHW-XIAO-Series#23] [Wish List]
Please make some add-ons with sensors:
- bme280
- light sensor
- accelerate sensor
- compas
- imu (gyro x-y-)

And some more

**Comment by meilily** [comment]
Hi @usermctech, thanks for the suggestion. All these sensors are available in our Grove Sensor lines, which can be connected to via either a [Grove Shield For Seeed Studio XIAO](https://www.seeedstudio.com/Grove-Shield-for-Seeeduino-XIAO-p-4621.html) or a [Seeed Studio XIAO Expansion Board](https://www.seeedstudio.com/Seeeduino-XIAO-Expansion-board-p-4746.html). Do you mean to make these sensors in the form facter of XIAO?

P.S. Acutually for IMU, we have a 6-axis IMU in XIAO nRF52840 Sense: https://www.seeedstudio.com/Seeed-XIAO-BLE-Sense-nRF52840-p-5253.html

Another P.S. We'll have a new XIAO coming up on Oct 30, 2024, which will also comes with a IMU too. Stay tuned by subscribe to our newsletter: https://mailchi.mp/seeed/xiao

**Comment by usermctech** [comment]
Thank you for the reply, but I want to have my device as small as possible, so grove will be too big

Wysłano z programu Outlook dla systemu Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by meilily** [reply]
Thanks for sharing. A big news for you. One of the community members has made part of your dreams come true via our [Fusion Co-Create Program](https://www.seeedstudio.com/co-create.html). Meet XIAO Logger HAT, a tiny temperature, humidity and light add-on module for XIAO with RTC and battery voltage monitoring. This tiny cool creation is designed by [Westlake University's Marcel](https://github.com/potblitd/XIAO-log/tree/main). Learn more about the product here: https://www.seeedstudio.com/XIAO-LOG-p-6341.html

<img width="598" alt="截屏2024-12-31 17 53 27" src="https://github.com/user-attachments/assets/4d2c0879-9421-42fa-9444-f49dcfb99bf7" />
